SU2_PY/pySU2/pySU2.i

```diff
@@ -1,22 +1,62 @@
-// SWIG input file of the 'SU2Solver' API module
+/*
+################################################################################
+#
+# \file pySU2.i
+# \brief Configuration file for the Swig compilation of the Python wrapper.
+# \author D. Thomas
+# \version 4.3.0 "Cardinal"
+#
+# SU2 Lead Developers: Dr. Francisco Palacios (Francisco.D.Palacios@boeing.com).
+#                      Dr. Thomas D. Economon (economon@stanford.edu).
+#
+# SU2 Developers: Prof. Juan J. Alonso's group at Stanford University.
+#                 Prof. Piero Colonna's group at Delft University of Technology.
+#                 Prof. Nicolas R. Gauger's group at Kaiserslautern University of Technology.
+#                 Prof. Alberto Guardone's group at Polytechnic University of Milan.
+#                 Prof. Rafael Palacios' group at Imperial College London.
+#                 Prof. Edwin van der Weide's group at the University of Twente.
+#                 Prof. Vincent Terrapon's group at the University of Liege.
+#
+# Copyright (C) 2012-2016 SU2, the open-source CFD code.
+#
+# SU2 is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# SU2 is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with SU2. If not, see <http://www.gnu.org/licenses/>.
+#
+################################################################################
+*/
 
 %feature("autodoc","1");
 
 %module(docstring=
-"'SU2Solver' module",
+"'WrapSU2' module",

```

Comment:
The best name choice is still unclear for me.. ideally it would just be SU2, but the old Python framework already uses this. Perhaps "pySU2" instead? Any suggestions?